changelog/unreleased/issue-4516

```diff
@@ -0,0 +1,7 @@
+Bugfix: Do not try to check password on command line autocomplete
+
+The command line autocompletion previously tried to load the repository
+password. Now, this step gets skipped.
```

Comment:
Suggest `The command line autocompletion previously tried to load the repository password. This could cause the autocompletion not to work. Now, this step is skipped.`

Reply:
Thanks. I've applied the suggestions.